### ISSUE TITLE
Anyone trained a batch-normalized vgg-verydeep model on Imagenet? 

### ISSUE THREAD

**Issue by peiyunh** [vlfeat/matconvnet#235]
I'm wondering how that performs and if there is any publicly available model?

**Comment by albanie** [comment]
The *VGG very deep* architecture trained on Imagenet with batch normalization is available on the pretrained models [download page](http://www.vlfeat.org/matconvnet/pretrained/).
